### PR TITLE
feat: upgrade aspects-dbt to v3.26.0

### PR DESCRIPTION
**Pull request by Ian2012**
### Description

This PR upgrades aspects-dbt to v3.26.0 and makes the proper changes to the datasets.

### Note

The dataset used here is the same as the one defined on aspects-dbt but includes the implemented Jinja optimization.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by openedx-webhooks** [comment]
@Ian2012 Even though your pull request wasn’t merged, please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/825&entry.1484655318=https://github.com/Ian2012&entry.752974735=2024-05-23+20:56&entry.1917517419=2024-05-24+12:04&entry.2133058324=No) so we can improve your experience in the future.

File: tutoraspects/templates/openedx-assets/queries/int_problem_results.sql [hunk 1]
```diff
@@ -1,4 +1,76 @@
...
+        select
+            org,
+            course_key,
+            problem_id,
+            actor_id,
+            max(last_attempt_at) as last_attempt_at
+        from {{ ASPECTS_XAPI_DATABASE }}.responses
+        where actor_id not in (select distinct actor_id from successful_responses)
+        group by org, course_key, problem_id, actor_id
+        {% include 'openedx-assets/queries/common_filters.sql' %}
```
Comment (bmtcril) [inline tutoraspects/templates/openedx-assets/queries/int_problem_results.sql:25]: This needs to go above the group by

File: tutoraspects/templates/openedx-assets/queries/int_problem_results.sql [hunk 2]
```diff
@@ -1,4 +1,76 @@
...
+    ),
+    -- for all learners who did not submit a successful response,
+    -- find the timestamp of the most recent unsuccessful response
+    unsuccessful_responses as (
+        select
+            org,
+            course_key,
+            problem_id,
+            actor_id,
```
Comment (bmtcril) [inline tutoraspects/templates/openedx-assets/queries/int_problem_results.sql:20]: I think you need to coerce these to strings, here and above?

**Review by bmtcril — APPROVED** [review]
Good to go once it passes CI!

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-aspects&entry.867055334=https://github.com/openedx/tutor-contrib-aspects/pull/825&entry.1484655318=https://github.com/Ian2012&entry.752974735=2024-05-23+20:56&entry.1917517419=2024-05-24+20:33&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**